applications/nrf_desktop/src/modules/ble_bond.c

```diff
@@ -231,6 +231,7 @@ static u8_t next_peer_id(u8_t id)
 	id++;
 	BUILD_ASSERT(TEMP_PEER_ID == (CONFIG_BT_MAX_PAIRED - 1));
 	BUILD_ASSERT(DONGLE_PEER_ID <= TEMP_PEER_ID);
+	__ASSERT_NO_MSG(id <= DONGLE_PEER_ID);
 	if (id == DONGLE_PEER_ID) {
```

Comment:
I would changed this condition to ">=" - to be defensive. In release environment you are on a safe side, in debug, you have an assertion to catch problematic situations. What more that change does not affect performacne.

Reply:
Done.